main/plots.py

```diff
@@ -99,22 +101,225 @@ def create_capacity_planning_plot(start_date: datetime, end_date: datetime) -> f
 
     Returns:
-        Bokeh figure containing timeseries data.
+        Bokeh Row layout with timeseries data plot and CheckboxGroup widgets.
+
     """
-    effort_timeseries = timeseries.get_effort_timeseries(start_date, end_date)
+    # Get effort for each project and user
+    PROJECTS = ["proCAT 1", "proCAT 2", "proCAT 3", "proCAT 4"]
+    USERS = ["User 1", "User 2", "User 3", "User 4", "User 5"]
+    project_colours = ["firebrick", "orange", "green", "blue"]
+    user_colours = ["purple", "brown", "pink", "cyan", "magenta"]
+
+    # Total effort and capacity across all projects
+    total_effort_timeseries = timeseries.get_effort_timeseries(start_date, end_date)
     capacity_timeseries = timeseries.get_capacity_timeseries(start_date, end_date)
+
     traces = [
+        {"timeseries": capacity_timeseries, "colour": "navy", "label": "Capacity"},
         {
-            "timeseries": effort_timeseries,
-            "colour": "firebrick",
-            "label": "Project effort",
+            "timeseries": total_effort_timeseries,
+            "colour": "black",
+            "label": "Total effort",
         },
-        {"timeseries": capacity_timeseries, "colour": "navy", "label": "Capacity"},
     ]
+
+    # Add effort for each project
+    for i, project in enumerate(PROJECTS):
+        project_effort_timeseries = timeseries.get_project_effort_timeseries(
+            start_date, end_date, project
+        )
+        traces.append(
+            {
+                "timeseries": project_effort_timeseries,
+                "colour": project_colours[i],
+                "label": project,
+            }
+        )
+
+    # Add effort for each user
+    for i, user in enumerate(USERS):
+        user_effort_timeseries = timeseries.get_user_effort_timeseries(
+            start_date, end_date, user
+        )
+        traces.append(
+            {
+                "timeseries": user_effort_timeseries,
+                "colour": user_colours[i],
+                "label": user,
+            }
+        )
+
     plot = create_timeseries_plot(
         title="Project effort and team capacity over time", traces=traces
     )
-    return plot
+
+    plot.yaxis.axis_label = "Effort (hours)"
+
+    plot.width = 650  # Reduce the width to fit the checkbox section
+
+    # CheckboxGroup for project selection
+    project_labels = ["Capacity", "Total effort", *PROJECTS]
+    project_checkbox_group = CheckboxGroup(
+        labels=project_labels,
+        active=[0, 1],  # Default to show capacity and total effort only
+        width=180,  # width of the checkbox group fr projects
+    )
+
+    # CheckboxGroup for user selection
+    user_labels = ["Capacity", "Total effort", *USERS]
+    user_checkbox_group = CheckboxGroup(
+        labels=user_labels,
+        active=[0, 1],  # Default to show capacity and total effort only
+        width=180,  # width of the checkbox group for users
+    )
```

Comment:
Not sure if CheckboxGroup is the right choice. It is nice because you can see all the options at once, but the list might grow a lot at times. Could you try `MultiSelect`?